Frontend/ui-driver/src/Screens/RideSummaryScreen/View.purs

```diff
@@ -0,0 +1,923 @@
+  in
+  linearLayout
+  [ height WRAP_CONTENT
+  , width WRAP_CONTENT
+  , background Color.green900
+  , gravity CENTER
+  , padding $ Padding 12 8 12 8 
+  , cornerRadii  (Corners 8.0 true false true false)
+  , visibility $ boolToVisibility $ not state.props.throughBanner
+  ]
+  [ textView $ 
+    [ text ( getString PICK_UP <>" : " <>  (JB.fromMetersToKm (distanceToPickup)) <> getString AWAY)
+    , color $ Color.white900
+    ] <> FontStyle.subHeading2 LanguageStyle
+  ]
+
+
+
+includedChargesCard :: RideSummaryScreenState -> (Action -> Effect Unit) -> forall w . PrestoDOM (Effect Unit) w
+includedChargesCard state push = 
+    PrestoAnim.animationSet
+    ([ Anim.fadeInWithDuration 200 true ] <>
+    (if os == "IOS" then []
+    else [ Anim.listExpandingAnimation $ estimateExpandingAnimationConfig (0) (0.0) true ]))
+  $ linearLayout
+    [ width MATCH_PARENT
+    , height WRAP_CONTENT
+    , orientation VERTICAL
+    ]
+    [ includedChargesBox state push 
+    , linearLayout
+        [width MATCH_PARENT
+        , height $ V 20
+        ][]
+    , includedChargesFooter state push 
+    ]
+
+
+includedChargesBox :: RideSummaryScreenState -> (Action -> Effect Unit) -> forall w . PrestoDOM (Effect Unit) w
+includedChargesBox state push =  
+    let 
+      (BookingAPIEntity entity) = state.data.rideDetails
+      dummyFare = RateCardItem{ price : 0, priceWithCurrency : {amount:0.0, currency:"INR"}, title : UNPLANNED_PER_KM_CHARGE }
+      (RateCardItem fare) = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == SERVICE_CHARGE) state.data.fareDetails) !! 0) 
+      (RateCardItem governmentFare) = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == GOVERNMENT_CHARGE) state.data.fareDetails) !! 0) 
+      (RateCardItem minFare) = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == MIN_FARE) state.data.fareDetails) !! 0) 
+      (RateCardItem perHourFare) = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == PER_HOUR_CHARGE) state.data.fareDetails) !! 0) 
+      (RateCardItem perMinFare) = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == PER_MINUTE_CHARGE) state.data.fareDetails) !! 0) 
+      (RateCardItem unplannedKMFare) = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == UNPLANNED_PER_KM_CHARGE) state.data.fareDetails) !! 0) 
+      (RateCardItem perHourKmFare) = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == PER_HOUR_DISTANCE_KM) state.data.fareDetails) !! 0) 
+      (RateCardItem plannedKMFare) = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == PLANNED_PER_KM_CHARGE) state.data.fareDetails) !! 0) 
+      (RateCardItem deadKMFare) = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == DEAD_KILOMETER_FARE) state.data.fareDetails) !! 0) 
+      (RateCardItem waitPerMinFare) = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == WAITING_CHARGE_PER_MIN) state.data.fareDetails) !! 0) 
```

Comment:
replace this code with the following : 

`let
  (BookingAPIEntity entity) = state.data.rideDetails
  dummyFare = RateCardItem { price: 0, priceWithCurrency: { amount: 0.0, currency: "INR" }, title: UNPLANNED_PER_KM_CHARGE }

  getFare :: String -> RateCardItem
  getFare title = fromMaybe dummyFare ((filter (\(RateCardItem x) -> x.title == title) state.data.fareDetails) !! 0)

  fare = getFare SERVICE_CHARGE
  governmentFare = getFare GOVERNMENT_CHARGE
  minFare = getFare MIN_FARE
  perHourFare = getFare PER_HOUR_CHARGE
  perMinFare = getFare PER_MINUTE_CHARGE
  unplannedKMFare = getFare UNPLANNED_PER_KM_CHARGE
  perHourKmFare = getFare PER_HOUR_DISTANCE_KM
  plannedKMFare = getFare PLANNED_PER_KM_CHARGE
  deadKMFare = getFare DEAD_KILOMETER_FARE
  waitPerMinFare = getFare WAITING_CHARGE_PER_MIN
  waitPerMinChargeFare = getFare WAITING_CHARGE_RATE_PER_MIN
  nightShiftFare = getFare NIGHT_SHIFT_CHARGE
  (CTA.TripCategory tripCategory) = entity.tripCategory`